constraint_utils/constraint_basics.py

```diff
@@ -355,6 +395,14 @@ def get_proportion_observed_by_coverage(exome_ht: hl.Table, context_ht: hl.Table
     freq_index = exome_ht.freq_index_dict.collect()[0][dataset]
 
     def keep_criteria(ht):
+        """
+        Generate expression to keep pass variants with an allele count greater than 0.
+        
+        If impose_high_af_cutoff_upfront is True, only keep variants with an AF less than the af_cutoff (default is 0.001).
+        
+        :param ht: Table of variants that have a defined key in the context ht
+        :return: BooleanExpression indicating if the row should be kept        
+        """
```

Comment:
```suggestion
        :param ht: Table of variants that have a defined key in `context_ht`.
        :return: BooleanExpression indicating if the row should be kept.
        """
```

Reply:
Again I don't think that this is actually a Table, it's `exome_ht`  indexed by the `context_ht` key